docs/userguide.md

```diff
@@ -930,3 +932,21 @@ only one instance is responsible for reconciliation. You can refer to the
 [helm chart configuration](https://github.com/kubernetes-sigs/kubefed/tree/master/charts/kubefed#configuration)
 to configure parameters for leader election to tune for your environment
 (the defaults should be sane for most environments).
+
+## Limitations
+### Immutable Fields
+Federation API does not implement immutable fields in the federated resource yet.
+
+A kubernetes resource field can be patched during runtime to change the resource
+specification. But the immutable field is not allowed to be modified after the
+resource was created.
+
+For federated resource, `spec.template` defines target resource specification.
```

Comment:
s/For/For a/
s/target resource specification/the resource specification common to all clusters/